cdf_dump/README.md

```diff
@@ -0,0 +1,186 @@
+# cdf_dump CLI tool
+
+The cdf_dump utility is a simple CLI wrapper for [Cognite Spark Data Source](https://github.com/cognitedata/cdp-spark-datasource)
+to simplify copying of large amounts data from CDF.
+
+## Installation
+
+Download the cdf_dump.jar from https://github.com/cognitedata/cdp-spark-datasource/releases and run it from command line: `java -jar cdf_dump.jar`. The cdf_dump.jar file contains all dependencies, including Apache Spark, so it's bit large, but should not require anything except the JVM on your OS.
```

Comment:
`https://github.com/cognitedata/cdp-spark-datasource/releases` maybe make this a link, exposing it as a clean string